src/js/helpers/Util.js

```diff
@@ -132,6 +132,22 @@ var Util = {
       b != null &&
       a.length === b.length &&
       a.every((v, i) => v === b[i]);
+  },
+  compareProperties: function (a, b, ...keys) {
+    return keys.reduce((memo, key) => {
+      var aVal = a[key];
+      var bVal = b[key];
+      var typeOfA = typeof(aVal);
+      if (Array.isArray(aVal)) {
+        return memo && this.isArrayEqual(aVal, bVal);
+      }
+      if (this.isObject(aVal) && this.isObject(bVal)) {
+        return memo
+          && this.isArrayEqual(Object.keys(aVal), Object.keys(bVal))
+          && this.isArrayEqual(Object.values(aVal), Object.values(bVal));
+      }
+      return memo && aVal === bVal;
+    }, true);
```

Comment:
.every() returns false, if one check is falsely, if _every_ check is truthy it returns true.